Download.Mvc/Client/components/Search.tsx

```diff
@@ -49,7 +56,10 @@ export class Search extends React.Component<IProps, any> {
         return (
             <div>
                 <SearchBar />
-                <h3>Searching for: {decodeURIComponent(this.props.match.params.query)}</h3>
+                {this.props.match.params.query &&
+                    <h3>Searching for: {decodeURIComponent(this.props.match.params.query)}</h3>}
+                {!this.props.match.params.query &&
+                    <h3>Search for: </h3>}
```

Comment:
@srkirkland if I don't add it, it renders "Searching for: undefined" which bugged me. But no, I didn't mean to change the language whoops